spec/services/participant_units_builder_spec.rb

```diff
@@ -47,9 +47,9 @@
     context 'when a user would have multiple roles' do
       let(:participants_json_path) { 'spec/fixtures/extracted_participations_with_multiple_roles.json' }
 
-      it 'does not create a warning' do
+      it 'creates a warning' do
```

Comment:
this might be a bit misleading, it was just a hidden error before, since we did not say 
```rb
 allow(Rollbar.configuration).to receive(:enabled).and_return(true)
 ```
 but it's expected, that it still creates a warning